flamedisx/block_source.py

```diff
@@ -32,6 +32,9 @@ class Block:
     #: _calculate_dimsizes_special().
     bonus_dimensions: ty.Tuple[ty.Tuple[str, bool]] = tuple()
 
+    #: inner_dimensions which take non-integer values
```

Comment:
I think all the changes in this file duplicate the PR #258, so we should merge that in first

Reply:
Was merged on Jan 19